source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import LinkIcon from 'octicon/link.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+async function repoListInit(): Promise<void> {
+	for (const repo of select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]')) {
+		repo.after(
+			<a
+				href={`https://${repo.textContent!.trim()}`}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{" "}<LinkExternalIcon/>
+			</a>
+		);
+	}
+}
+
+async function repoInit(): Promise<void | false> {
+	const githubIoLink = getRepositoryInfo().name!;
+
+	if (select.exists('.octicon-link', await elementReady('.BorderGrid-cell'))) {
+		return false;
+	}
+
+	select('.BorderGrid-cell')!.append(
```

Comment:
Only do operations when necessary:

```suggestion
	if (select.exists('.octicon-link', await elementReady('.BorderGrid-cell'))) {
		return false;
	}

	const githubIoLink = getRepositoryInfo().name!;
	select('.BorderGrid-cell')!.append(
```

Reply:
done